assets/init

```diff
@@ -721,7 +721,7 @@ chmod 700 ${GITLAB_DATA_DIR}/.ssh
 chmod 600 ${GITLAB_DATA_DIR}/.ssh/authorized_keys
 chown -R git:git ${GITLAB_DATA_DIR}/.ssh
 
-appStart () {
+appInit () {
```

Comment:
@aforward I think you should modify the `appInit` function to `return 1` on failures. i.e. replace the `exit 1` with `return 1` and end the function with `return 0`